vendordeps/Phoenix6.json

```diff
@@ -0,0 +1,339 @@
+{
+  "fileName": "Phoenix6.json",
+  "name": "CTRE-Phoenix (v6)",
+  "version": "24.0.0-beta-4",
+  "frcYear": 2024,
+  "uuid": "e995de00-2c64-4df5-8831-c1441420ff19",
+  "mavenUrls": [
+    "https://maven.ctr-electronics.com/release/"
+  ],
+  "jsonUrl": "https://maven.ctr-electronics.com/release/com/ctre/phoenix6/latest/Phoenix6-frc2024-beta-latest.json",
+  "conflictsWith": [
+    {
+      "uuid": "3fcf3402-e646-4fa6-971e-18afe8173b1a",
+      "errorMessage": "The combined Phoenix-6-And-5 vendordep is no longer supported. Please remove the vendordep and instead add both the latest Phoenix 6 vendordep and Phoenix 5 vendordep.",
```

Comment:
Do we still use Phoenix 5 devices? If so, the docs still state the `Phoenix6And5` vendordep still exists: https://pro.docs.ctr-electronics.com/en/latest/docs/installation/installation.html#online
I'd try removing `Phoenix5.json` and `Phoenix6.json`, adding the combined JSON, and see whether it works, or this error message is real.

Reply:
I don't want to use phoenix 5, but I want to leave it just in case. Why do you want me to use the merged json?

Reply:
Easier to maintain and update one file instead of two.

Reply:
I changed my mind. IMO it's best to delete the Phoenix 5 file, right now there's a risk of importing the same class from Phoenix 5 instead of Phoenix 6. Removing this file will eliminate this risk, we can always add it back if needed.